src/twisted/internet/endpoints.py

```diff
@@ -1466,6 +1470,10 @@ def _parseSSL(
         dhParameters=dhParameters,
         **kw,
     )
+
+    if cipher:
+        cipher_bytes = cipher.replace(",", ":").encode("ascii")
+        cf.getContext().set_cipher_list(cipher_bytes)
```

Comment:
Twisted's coding standard is slightly different than PEP8; you should be using `camelCase` rather than `snake_case` for local variables.

```suggestion
        cipherBytes = cipher.replace(",", ":").encode("ascii")
        cf.getContext().set_cipher_list(cipherBytes)
```